client/ayon_core/cli.py

```diff
@@ -258,6 +269,21 @@ def _set_addons_environments():
         os.environ.update(env)
 
 
+def _add_addons(addons_manager):
+    """Modules/Addons can add their cli commands dynamically."""
+    log = Logger.get_logger("CLI-AddModules")
```

Comment:
The logger name feels a bit weird?

It's adding CLIs, in a function named `_add_addons` yet the logger name references to Modules? Whih is it? :)

Should we drop naming this "modules" and just refer to them as addons?

Reply:
Why you just don't add suggestion?
```suggestion
    log = Logger.get_logger("CLI-AddAddons")
```